forms/pdf_form_fields_rotations.go

```diff
@@ -0,0 +1,139 @@
+/*
+ * Add images to a PDF file, one image per page.
```

Comment:
@sampila maybe update the description and indicate how this is related to forms?

Reply:
Updated at 103bfc2cb8d0ec2ff69315f3023f9df9f3132268